src/test/java/unicash/logic/commands/GetTotalExpenditureCommandTest.java

```diff
@@ -69,6 +69,41 @@ public void execute_multipleMonthsOnly_filtersOnlySelectedMonths() throws Comman
         }
     }
 
+    @Test
+    public void execute_expenseWithoutCategoryWithCategoryFilter_notIncludedInFilter() throws CommandException {
+        var model = getModel();
+        // This transaction does not contain any categories even if it's in August
```

Comment:
What's this comment supposed to mean? The transaction contains no categories in spite of its being dated in August?

Reply:
Yeap